src/v/raft/group_manager.cc

```diff
@@ -64,9 +77,13 @@ group_manager::group_manager(
 ss::future<> group_manager::start() {
     co_await _heartbeats.start();
     co_await _recovery_scheduler.start();
+    _metrics_timer.arm(_metrics_interval_jitter.next_jitter_duration());
```

Comment:
nit: use arm_periodic() ? (get rid of the other arm)

Reply:
we have a jitter this is why i was using arm, but as Alexey suggests maybe jitter is not needed